test/_unit.js

```diff
@@ -209,6 +209,16 @@ suite('Lexer', () => {
         if (k !== 'default') exports[k] = externalǽ[k];
       });
 
+      let external5 = require('e5');
+      let external6 = require('e6');
+      Object.keys(external5).forEach(function (k) {
+        if (k !== 'default' && !Object.hasOwnProperty.call(exports, k)) exports[k] = external5[k];
+      });
+
+      Object.keys(external6).forEach(function (k) {
+        if (k !== 'default' && !external6.hasOwnProperty(k)) exports[k] = external6[k];
```

Comment:
It's `exports.hasOwnProperty` not `external6.hasOwnProperty` 😅

Reply:
Thanks for spotting this, wish we'd got this on the first review! Posted https://github.com/guybedford/cjs-module-lexer/pull/59.

Reply:
@sodatea I've released the update here in 1.2.2 (https://github.com/guybedford/cjs-module-lexer/commit/a96e7b52b99d39fee25d69f356905161d14a15ed) and am starting the process to backport the updates across all Node.js release lines for the releases next month. If you are able to verify the fix separately that would be a huge help to ensure there are no further regressions here.

Reply:
Thanks! I think it works correctly now.